### ISSUE TITLE
Formula image is blank

### ISSUE THREAD

**Issue by bhbr** [bhbr/MQKB#17]
![Screenshot](https://files.slack.com/files-pri/T7ES7S3KQ-F7G2AV0FM/image_uploaded_from_ios.jpg)

**Comment by bhbr** [comment]
iPhone 6 on iOS 10.3.2, full access granted

**Comment by jdotmac112** [comment]
I'm having this issue as well. iPhone 7 plus with iOS 10.3.3. The copy button doesn't seem to copy the formula as the image is blank when pasted.

**Comment by bhbr** [comment]
But there is an image pasted, right? It's just all white, rather than no image at all?

**Comment by jdotmac112** [comment]
@bhbr That's correct. The image sends, can be selected, but simply displays white.

**Comment by bhbr** [comment]
Still present in v0.4

**Comment by bhbr** [comment]
Can you export the formula as TeX? (long press the copy button)

Also, what happens if you haven't typed anything? Does it still paste a white image or what was previously in the clipboard?

**Comment by bhbr** [comment]
This might be the issue: https://stackoverflow.com/questions/26843804/i-cant-capture-the-screenshot-of-a-wkwebview

**Comment by bhbr** [comment]
@jdotmac112 I'd like to run a few test versions by you. Do you know how to sideload apps via XCode?

**Comment by ghost** [comment]
@bhbr I would be happy to test some other versions for you over the weekend. I have XCode installed but never used that functionality. I'm sure I can figure it out. Do you still have my email?
